### ISSUE TITLE
how to freeze "updatedAt" column when update record

### ISSUE THREAD

**Issue by king804** [Automattic/mongoose#6980]
same as title

i would like to lock the updatedAt when some function is update the query

**Comment by vkarpov15** [comment]
Can you please clarify what you mean by freeze? Do you want to skip incrementing updatedAt for a single update? Or prevent all future changes to updatedAt?

**Comment by king804** [comment]
yes...i want to to skip incrementing updatedAt for a special update.
for example...
case 1. user login action ->update the "updatedAt" and update other field
case 2. user logout action -> don't update the updateAt but also update the other field

how can i achieve that?  


Many thanks!

**Comment by vkarpov15** [comment]
Currently you would have to use the underlying MongoDB collection's updateOne() to skip timestamps:

```javascript
MyModel.collection.updateOne({ _id }, { $set: { answer: 42 } }); // Won't update timestamps
```

We'll add an  option for this.
